views/js/GoogleAnalyticActionLib.js

```diff
@@ -156,7 +157,7 @@ var GoogleAnalyticEnhancedECommerce = {
 
 		//this.add(Product);
 		ga('ec:setAction', 'purchase', Order);
-		ga('send', 'pageview', {
+		ga('send', 'event','Transaction','purchase', {
```

Comment:
Regarding the [Google Analytics documentation](https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#transaction], the pageview is an expected step. Even if we can call this function in the BackOffice, the part in ```hookBackOfficeHeader()``` is supposed to send the transaction which have failed on the BO.